cirq-core/cirq/ops/matrix_gates.py

```diff
@@ -106,8 +110,12 @@ def _unitary_(self) -> np.ndarray:
     def _circuit_diagram_info_(
         self, args: 'cirq.CircuitDiagramInfoArgs'
     ) -> 'cirq.CircuitDiagramInfo':
+        qLen = len(self._qid_shape)
```

Comment:
don't use camel case, we always use snake case
```suggestion
        q_len = len(self._qid_shape)
```

but also, I'd rather call it `n_qubits`

Reply:
My bad.  Force of habit.  I'll update it.